src/modules/settings/LanguagePage.tsx

```diff
@@ -0,0 +1,41 @@
+import { Box, Container, List } from '@mui/material';
+import { HalfTypeNavigation } from '../../shared-components/Navigation/TopNavigation/HalfTypeNavigation';
+import { colors } from 'src/styles/colors';
+import { Language } from 'src/types/Language';
+import { ListItemSelect } from '../../shared-components/Card/ListItems/ListItemSelect';
+import { useState } from 'react';
+import { useUserDispatch } from 'src/context/UserContext';
+import { UserActionType } from '../../context/UserContext';
+
+export const LanguagePage = () => {
+  const [selectedLanguage, setSelectedLanguage] = useState<Language | null>(
```

Comment:
Same as with currency, access and modify data directly from Context.

Reply:
fixed